### PR TITLE
Add "loop" parameter to playSound

### PR DESCRIPTION
**Pull request by islemaster**
Updates the signature of the playSound command from `playSound(url:string)` to `playSound(url:string[, loop:boolean])`, along with the expected new functionality.

Requested in [this spec](https://docs.google.com/document/d/11mpYgmomALyAr53BQl2Ufx0ZYXoMAswqlQBA0aRuNag/edit#).

### CODE REVIEW

File: apps/src/lib/util/audioApi.js [hunk 1]
```diff
@@ -70,6 +71,7 @@ export const commands = {
       }
       studioApp.cdoSounds.playURL(url, {
         volume: 1.0,
+        loop: !!opts.loop,
```
Comment (islemaster) [inline apps/src/lib/util/audioApi.js:74]: This is the line where the magic actually happens - the core sound system already had this capability, we just weren't exposing the parameter.

**Comment by caleybrock** [comment]
tests haven't finished running yet, but any tests to update for the sound api?

**Comment by islemaster** [comment]
I could go ahead and add tests that verify matching signatures across the various configurations - that sounds helpful, actually, I'll do that.  There are currently no unit tests for the actual sound API behavior - I'm hoping to fix that on my next iteration, when I build a layer for handling more complex simultaneous playback with better support for stubbing the core audio interface.

**Comment by caleybrock** [comment]
👍

**Pull request merged**